### PR TITLE
[IMP] accounting: update the follow-up information

### PR DESCRIPTION
**Pull request by Donapi**
Task ID: 2735445
https://www.odoo.com/web#id=2735445&cids=1&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/finance/accounting/receivables/customer_payments/followup.rst [hunk 1]
```diff
@@ -1,70 +1,67 @@
...
+Debtor's trust level
+--------------------
 
-A follow-up plan with several actions is available by default in Odoo,
-but you can customize it any way you want. Depending on a specific
-number of overdue days, plan to send an email, a letter or to undertake
-a manual action. You can also edit the template used for the statement
-according to the stage of the process.
+To know whether a customer usually pays late or not, you can set a trust level on them by marking
+them as good, normal, or bad debtors on the follow-up reports. Just click on the button next to the
```
Comment (aim-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/followup.rst:56]:
```suggestion
them as good, normal, or bad debtor on the follow-up report. Just click on the bullet next to the
```

File: content/applications/finance/accounting/receivables/customer_payments/followup.rst [hunk 2]
```diff
@@ -1,70 +1,67 @@
...
+.. image:: media/followup01.png
     :align: center
 
-.. tip::
-    If you would like to get a reminder before the actual due date,
-    set a negative number of due days.
+Send reminders in batch
+-----------------------
+
+To make your follow-up process more manageable, you can send reminder emails in batches from your
```
Comment (aim-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/followup.rst:65]:
```suggestion
To make your follow-up process even smoother, you can send reminder emails in batches from the
```

File: content/applications/finance/accounting/receivables/customer_payments/followup.rst [hunk 3]
```diff
@@ -1,70 +1,67 @@
...
+customer's name to do so.
 
-.. image:: media/followup03.png
+.. image:: media/followup01.png
     :align: center
 
-.. tip::
-    If you would like to get a reminder before the actual due date,
-    set a negative number of due days.
+Send reminders in batch
```
Comment (aim-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/followup.rst:62]:
```suggestion
Send reminders in batches
```

File: content/applications/finance/accounting/receivables/customer_payments/followup.rst [hunk 4]
```diff
@@ -1,70 +1,67 @@
...
+-----------------------
+
+To make your follow-up process more manageable, you can send reminder emails in batches from your
+Follow-up Reports page. Select all the reports you would like to process, click on Action, and then
```
Comment (aim-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/followup.rst:66]:
```suggestion
Follow-up Reports page. Select all the reports you would like to process, click on the *Action* gear icon, and select
```

File: content/applications/finance/accounting/receivables/customer_payments/followup.rst [hunk 5]
```diff
@@ -1,70 +1,67 @@
...
+on Process Follow-ups.
```
Comment (aim-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/followup.rst:67]:
```suggestion
*Process Follow-ups*.
```

**Review by aim-odoo — CHANGES_REQUESTED** [review]
@Donapi review done ;)

File: content/applications/finance/accounting/receivables/customer_payments/followup.rst [hunk 6]
```diff
@@ -1,70 +1,67 @@
...
-needs your attention.
+A follow-up plan, with several actions, is available by default in :menuselection:`Accounting -->
+Configuration --> Follow-up Levels`, but you can customize it any way you want.
 
-Otherwise, if now is not the time for a reminder, click on *Remind me
-later*. You will get the next report according to the *Next Reminder
-Date* set on the statement.
+Depending on the number of overdue days you decide, you can plan to print a letter, send an email,
+an SMS message, or a letter, to follow up on with your customer. It is also possible to
+automatically send a reminder by checking the *Auto Execute* option
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/receivables/customer_payments/followup.rst:20]: Sentence updated as follow: Depending on the number of overdue days you set in the *Follow-up Action* form, you can plan to print a letter, send an email, an SMS message, or a letter to follow up with your customer. It is
also possible to automatically send a reminder by enabling the *Auto Execute* option.

File: content/applications/finance/accounting/receivables/customer_payments/followup.rst [hunk 7]
```diff
@@ -1,70 +1,67 @@
...
-A pdf document containing all of the follow-up letters for you to print
-will also be automatically generated.
+It is up to you to decide how to remind your customer. You can select *Print Letter*,
+*Send By Email*, *Send By Post*, *Send By SMS*. Then, click on the *Done* button to view the next
+follow up that needs your attention.
 
-.. image:: media/followup02.png
-    :align: center
+.. note::
+   * The contact information available on the invoice or on the contact is used to send the
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/receivables/customer_payments/followup.rst:41]: The contact information available on the invoice or on the contact form is used for? 
"used to send the reminder." isn't correct?

**Comment by Donapi** [comment]
Thanks for your review @aim-odoo !

File: content/applications/finance/accounting/receivables/customer_payments/followup.rst [hunk 8]
```diff
@@ -1,70 +1,67 @@
...
-batch from your follow-up reports page. Select all the reports you would
-like to process, click on *Action* and then on *Process Follow-ups*.
-A pdf document containing all of the follow-up letters for you to print
-will also be automatically generated.
+It is up to you to decide how to remind your customer. You can select *Print Letter*,
+*Send By Email*, *Send By Post*, *Send By SMS*. Then, click on the *Done* button to view the next
+follow-up that needs your attention.
 
-.. image:: media/followup02.png
-    :align: center
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/receivables/customer_payments/followup.rst:49]: I no longer see it...

File: content/applications/finance/accounting/receivables/customer_payments/followup.rst [hunk 9]
```diff
@@ -1,70 +1,67 @@
...
 .. tip::
-    To avoid sending too many reminders in a short period of time,
-    change the number of days calculated between each report by going to
-    :menuselection:`Accounting --> Configuration --> Settings --> Payment
-    Follow-up`.
+   To send a reminder before the actual due date is reached, set a negative number of due days.
 
-You can also set a trust level for your customers by marking them as
-bad, normal or good debtors on the follow-up reports.
+Follow-up Reports
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/receivables/customer_payments/followup.rst:25]: Le terme est en majuscule dans Odoo... 
image

[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/receivables/customer_payments/followup.rst:25]: du coup, majuscule ou minuscule? :)

**Review by jcs-odoo — APPROVED** [review]

**Comment by jcs-odoo** [comment]
Ping me once you've fixed the conflicts with the old images' paths.

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**